src/wymeditor/editor/base.js

```diff
@@ -2032,18 +2032,15 @@ WYMeditor.editor.prototype.paste = function (str) {
 WYMeditor.editor.prototype.insert = function (html) {
     // Do we have a selection?
     var wym = this,
-        selection = wym.selection(),
+        selection = wym.hasSelection() ? wym.selection() : false,
```

Comment:
[Style guide](http://wymeditor.readthedocs.org/en/latest/wymeditor_development/coding_standard.html)
doesn't mention it and it refers to the Opera style guide, right at the end, which does suggest use of it. And it is very common in modern JS.